### DISCUSSION TITLE
Github student pack for my account has ended too early then it should have

### DISCUSSION THREAD

**Discussion by MuhammadAliAsgher** [community/community#135252] [GitHub Education]
### Hi everyone,

I have obtained github student pack in July 2022
And should still be active until June 2025, according to the student ID card(image attached)

But last month, I got the email that my github student pack benefit has ended. (image attached)
Which is too early... and GitHub is starting to billing me if I wished to keep using the services.

I have tried to reverify my academic status according to this comment: "**https://github.com/orgs/community/discussions/42245#discussioncomment-4751715**",  but I am not able to do that as well. It is asking me to provide some other form or evidence other than my ID card to show affiliation with the university, but I don't have anything except my card (image attached)

I need help, thanks in advance.

Account Name: MH-11103 (https://github.com/MH-11103)

![1](https://github.com/user-attachments/assets/af6d2ed3-eb1c-4624-bc43-c1cb95b55dac)
![2](https://github.com/user-attachments/assets/4461f383-8b7a-4b50-a737-2619c92ede1a)
![3](https://github.com/user-attachments/assets/3b29b0db-323e-45ca-9066-f2ccbac33c3b)

**Comment by samus-aran** [comment]
Hi @MH-11103 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. 

- First things first, enable 2FA and update your Billing Information.

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Comment by davevad93** [comment]
Hi @MH-11103 ,

Student benefits are valid for 2 years, regardless of the expiration date of the ID card after which you'll need to re-verify by uploading documents that prove your current student status.

Your ID card doesn't show a current date, so maybe that's why they ask for more proof; when I've applied to get the benefits I initially uploaded my enrollment sheet with my details, school name and the academic year, but I was rejected because it wasn't enough proof for them, so in the end I uploaded a screenshot of my latest exam results showing a date in `DD/MM/YYYY` format and in the end I was approved.

The [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) state the following about the required student documentation:

- ***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***

So if possible try to provide some of the above documentation in addition to your ID card.
